src/main/java/io/mosip/digitalcard/websub/DigitalCardInstializer.java

```diff
@@ -0,0 +1,103 @@
+package io.mosip.digitalcard.websub;
+
+import io.mosip.digitalcard.util.DigitalCardRepoLogger;
+import io.mosip.digitalcard.util.Utility;
+import io.mosip.kernel.core.logger.spi.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.event.ApplicationReadyEvent;
+import org.springframework.context.ApplicationListener;
+import org.springframework.scheduling.concurrent.ThreadPoolTaskScheduler;
+import org.springframework.stereotype.Component;
+
+import java.time.Duration;
+import java.time.Instant;
+
+@Component
+public class DigitalCardInstializer implements ApplicationListener<ApplicationReadyEvent> {

```

Comment:
There is a spelling mistake here

Reply:
taken care

Reply:
I see "Instializer"  still misspelled, there are other places also this work is misspelled.

Reply:
DigitalCardInitializer have updated already